arquillian-ape-sql/standalone/dbunit/src/main/java/org/arquillian/ape/rdbms/dbunit/DbUnitOptions.java

```diff
@@ -0,0 +1,100 @@
+package org.arquillian.ape.rdbms.dbunit;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class DbUnitOptions implements Map<String, Object> {
```

Comment:
I agree but at the same time the only alternative I can think is creating a sub DSL, let's finish alpha2, and then let' me think an approach.